tests/test_generators/python/test_python_ifabsent_processor.py

```diff
@@ -374,6 +374,44 @@ def test_uri_default_value():
     )
 
 
+# If this function does not get fixture scoped to the module the
+# PythonIfAbsentProcessor instance suddenly has some attributes
+# coming from tests using the kitchen-sink example! Invalidating that
+# way the results of the test.
+@pytest.fixture(scope="module")
```

Comment:
this is a test, right? so it shouldn't be a fixture at all? I don't see how it interacts with the kitchen sink example here.

in any case it being a fixture meant it is not run as a test, note its absence (i did check out this branch and am looking at the code being there and it's in editable mode so i can cause tests to fail by throwing a spurious exception, so it's not my environment, just how fixtures work.

```
collected 15 items                                                                                            

tests/test_generators/python/test_python_ifabsent_processor.py::test_unimplemented_default_value[class_curie] PASSED [  6%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_unimplemented_default_value[class_uri] PASSED [ 13%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_unimplemented_default_value[slot_uri] PASSED [ 20%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_unimplemented_default_value[slot_curie] PASSED [ 26%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_unimplemented_default_value[default_range] PASSED [ 33%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_unimplemented_default_value[default_ns] PASSED [ 40%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_string_default_value PASSED        [ 46%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_boolean_default_value PASSED       [ 53%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_numeric_default_value PASSED       [ 60%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_time_default_value PASSED          [ 66%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_date_default_value PASSED          [ 73%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_datetime_default_value PASSED      [ 80%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_uri_default_value PASSED           [ 86%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_enum_default_value PASSED          [ 93%]
tests/test_generators/python/test_python_ifabsent_processor.py::test_bnode_default_value PASSED         [100%]

============================================= 15 passed in 0.77s ==============================================
```

Reply:
If i add a `pdb.set_trace()` call below the assignment and print it, i get what would be expected:

```yaml

id: ifabsent_tests
name: ifabsent_tests

prefixes:
  ex: https://example.org/
default_prefix: ex

classes:
  Student:
    attributes:

      - name: presence
        range: PresenceEnum
      - name: invalidPresence
        range: PresenceEnum

enums:
  PresenceEnum:
    permissible_values:
      Present:
        description: It's there.
      Missing:
        description: It's not there.
```

Reply:
This was a desperate late-hours try to fix an issue that was driving me nuts without a logical explanation I could find.  Apparently my laptop only needed a nap, since I cannot reproduce the issue anymore 🤷